client/src/state/editor/sortBlockMutation.js

```diff
@@ -56,7 +56,7 @@ const config = {
         }
 
         // Add it back to the full result
-        newData.readOnePage.ElementalAreaIfExists.Elements.edges = edges;
+        newData.readOnePage.Elements.edges = edges;
```

Comment:
Also here, should this be `readOneElementalArea`? I'm looking at the changes in the GraphQL query